bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/Target.scala

```diff
@@ -126,13 +125,14 @@ object Target {
   }
 
   object SiderealTarget {
+
+    def apply(name: String, coordinates: Coordinates, magnitudes: List[Magnitude]): SiderealTarget = SiderealTarget(name, coordinates, None, None, None, magnitudes)
```

Comment:
I'm not sure about this secondary factory method. I know you need it for some of your tests but you could define it locally (I have done this on my branch actually ... which will have a bunch of conflicts with this one).